.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "3rdparty/xbyak"]
 	path = 3rdparty/xbyak
-	url = ../xbyak.git
+	url = ../../RPCSX/xbyak.git
```

Comment:
can we move to an absolute path ? this breaks when building from a fork

Reply:
You are doing something wrong then